<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="eon-swapnili" name="atlantamarket_UAT.GlobalSearch_LineActions" tests="4" failures="0" timestamp="8 Jan 2024 12:48:49 GMT" time="153.057" errors="1">
  <testcase name="TS004_VerifyClickOnTotalProductsSeeAllLinkForLineTest" time="47.304" classname="atlantamarket_UAT.GlobalSearch_LineActions"/>
  <testcase name="TS002_VerifyClickOnLocationLinkForLineTest" time="24.240" classname="atlantamarket_UAT.GlobalSearch_LineActions"/>
  <testcase name="TS001_VerifyClickOnShownByExhibitorNameLinkForLineTest" time="25.152" classname="atlantamarket_UAT.GlobalSearch_LineActions"/>
  <testcase name="TS003_VerifyClickOnMatchingProductsSeeAllLinkForLineTest" time="56.361" classname="atlantamarket_UAT.GlobalSearch_LineActions">
    <error type="java.lang.AssertionError" message="expected [2] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [401]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at atlantamarket_UAT.GlobalSearch_LineActions.TS003_VerifyClickOnMatchingProductsSeeAllLinkForLineTest(GlobalSearch_LineActions.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- TS003_VerifyClickOnMatchingProductsSeeAllLinkForLineTest -->
</testsuite> <!-- atlantamarket_UAT.GlobalSearch_LineActions -->
